src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -2,21 +2,22 @@ package racingcar.domain
 
 import racingcar.view.OutputView
 
-const val MOVE_FORWARD = 4
-
 class Cars {
     private val cars = mutableListOf<Car>()
 
-    fun startRace(gameInputValue: GameInputValue, movingConditionStrategy: MovingConditionStrategy) {
+    fun make(numberOfCars: List<String>, movingConditionStrategy: MovingConditionStrategy) {
+        for (i in numberOfCars.indices) {
+            cars.add(Car(movingConditionStrategy, numberOfCars[i]))
+        }
```

Comment:
numberOfCars 네이밍이 잘못 된것 같습니다 ㅎㅎ 
```suggestion
        numberOfCars.map { Car(movingConditionStrategy, it) }
```
위 로직을 생성자로 정의하거나 init block 에 정의하는 방법으로 Cars 객체 생성을 더욱 깔끔하게 할수 있을것 같아요 !